Havoc Guide updates

ToppleTheNun opened the pull request:
### Description

<!-- A brief description of the changes made with this pull request.-->

Add Rotation section to Havoc Guide, include Furious Gaze on Eye Beam breakdown.

### Motivation

<!-- Why are you submitting this pull request?-->

A few changes were requested to help make the Havoc Guide more useful.

### Testing

<!-- How can the reviewer test your changes (if applicable)? -->

- Test report URL: `/report/6g4TqjXz9BH2ywDk/1-Mythic+Eranog+-+Kill+(2:15)/Hotbees/standard`
- Screenshot(s):
![image](https://user-images.githubusercontent.com/1672786/224574818-d8be676d-b9d2-435c-a698-244cab809589.png)
![image](https://user-images.githubusercontent.com/1672786/224574841-7ca3ad52-cb25-4bfb-82b3-55f9f56361e1.png)

emallson commented:
IMHO the sidebars here have too much text.

- Throw Glaive has a lot of repeating what the talents you're taking do. This can be trimmed down to something like "Throw Glaive is buffed by many talents in the spec tree, and is only worth casting if you take at least Soulrend and Furious Throws." Could optionally add a list of talents as spell links that buffed Throw Glaive to drive that point home.
- Essence Break's issue is mostly spelling out the cast sequence for the EB window. Would like to have a better pattern for solving this (Brew needs it now for EK, 2/3 mage specs want it, etc). For now I would probably make them collapsed by default that you can expand to view the ideal sequence.

ToppleTheNun commented:
Agreed, especially seeing how few checks there actually are for Essence Break and how little data there really is to show for Throw Glaive. I'll revisit how they're displayed.

ToppleTheNun commented:
Updated Throw Glaive section:
![image](https://user-images.githubusercontent.com/1672786/224754756-fff7c14c-9688-44c8-b9db-b9c6e5ba8c2a.png)

Updated Essence Break section:
![image](https://user-images.githubusercontent.com/1672786/224754880-fa245656-569d-4ab5-9815-53dbf677b041.png)

Updated Essence Break with both collapsed views open:
![image](https://user-images.githubusercontent.com/1672786/224754990-9494dcb7-b2b0-490a-b9c3-13c4eccb967b.png)
